src/stories/flash-card.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { Meta, StoryObj } from "@storybook/react";
```

Comment:
You need to rebase this branch to main as well. Please keep in mind when creating a branch locally to switch to main first.